prow/plugins/heart/heart.go

```diff
@@ -73,3 +83,34 @@ func handle(gc githubClient, log *logrus.Entry, ic github.IssueCommentEvent) err
 		ic.Comment.ID,
 		reactions[rand.Intn(len(reactions))])
 }
+
+func handlePR(gc githubClient, log *logrus.Entry, pre github.PullRequestEvent) error {
+	// Only consider newly opened PRs
+	if pre.Action != "opened" {
+		return nil
+	}
+
+	changes, err := gc.GetPullRequestChanges(pre.PullRequest)
+	if err != nil {
+		return err
+	}
+
+	// Smile at any change that adds to OWNERS files
+	ownersFileAdditions := false
+	for _, change := range changes {
+		_, filename := filepath.Split(change.Filename)
+		if filename == ownersFilename && change.Additions > 0 {
+			ownersFileAdditions = true
+			break
+		}
+	}
+
+	if ownersFileAdditions {
+		return gc.CreateIssueReaction(
```

Comment:
Add a log statement that we're taking an action. It doesn't need to be anything detailed, just something so that I can see when this code gets hit. For issue comments I have `log.Info("This is a wonderful thing!")`.